ee-feature-pack/common/pom.xml

```diff
@@ -1080,24 +1080,22 @@
 
         <dependency>
             <groupId>org.jboss.resteasy</groupId>
-            <artifactId>resteasy-jaxrs</artifactId>
-            <exclusions>
-                <exclusion>
-                    <groupId>*</groupId>
-                    <artifactId>*</artifactId>
-                </exclusion>
-            </exclusions>
+            <artifactId>resteasy-client-api</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.jboss.resteasy</groupId>
+            <artifactId>resteasy-core</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.jboss.resteasy</groupId>
+            <artifactId>resteasy-core-spi</artifactId>
         </dependency>
 
         <dependency>
             <groupId>org.jboss.resteasy</groupId>
             <artifactId>resteasy-spring</artifactId>
-            <exclusions>
```

Comment:
All transitive dependencies should be exclusively defined. It just makes it easier to exclude them all.

Reply:
That's what I was asking - the change here _removes_ the exclusion

Reply:
Ah sorry, I was didn't notice the pom. Yes, I moved all the exclusions to the main pom dependency so they wouldn't have to be excluded in the child pom's. See https://github.com/wildfly/wildfly/blob/af55fac516315dba297686ac825046448f6e1852/pom.xml#L8040-L8050

Reply:
OK, as long as these are always explicitly declared in child poms it should be safe (otherwise we risk https://issues.apache.org/jira/browse/MNG-5761)

Reply:
Interesting. I'd never seen that before. Most, if not all, WildFly dependencies are defined like that. AFAIK we have not seen an issue.